Commands/notify.py

```diff
@@ -72,22 +106,20 @@ async def wait_for_added_reactions(self, ctx, msg, guild_data, timeout):
     async def wait_for_removed_reactions(self, ctx, msg, guild_data, timeout):
         while True:
             try:
-                reaction, user = await ctx.bot.wait_for(
-                    "reaction_remove",
-                    check=lambda reaction, user: reaction.message.id == msg.id
-                    and not user.bot,
-                    timeout=30.0,
-                )
+                reaction, user = await ctx.bot.wait_for("reaction_remove", check=lambda reaction, user: reaction.message.id == msg.id and not user.bot, timeout=30.0,)
                 if reaction.custom_emoji:
                     reaction_emoji = str(reaction.emoji.id)
                 else:
                     reaction_emoji = reaction.emoji
                 for key, v in guild_data.notification_lists.items():
 
                     if reaction_emoji == v["emoji"]:
-
-                        msg_string = await guild_data.unsub_user(key, user.id)
-                        await ctx.send(msg_string)
+                        list_name = key
+                        succeeded = await guild_data.unsub_user(key, ctx.author.id)
+                        if succeeded:
+                            await ctx.send("Unubscribed <@" + str(ctx.author.id) + "> from " + list_name)
+                        else:
+                            await ctx.send("<@" + str(ctx.author.id) + ">, you are not subscribed to " + list_name)
```

Comment:
repetition of the code in command unsubscribe(). Would be more elegant to call self.unsubscribe() here to avoid repitition of the same logic.